src/main/java/com/github/tomakehurst/wiremock/http/ResponseDefinition.java

```diff
@@ -359,18 +414,26 @@ public boolean equals(Object o) {
             Objects.equals(proxyBaseUrl, that.proxyBaseUrl) &&
             fault == that.fault &&
             Objects.equals(transformers, that.transformers) &&
+            Objects.equals(transformerInstances, that.transformerInstances) &&
```

Comment:
not all the fields are currently included to the `equals`/`hashCode`, but I assume by default a fields should be